src/store/files.js

```diff
@@ -91,7 +91,8 @@ export default {
 				secretAccessKey: secretKey,
 				endpoint,
 				s3ForcePathStyle: true,
-				signatureVersion: "v4"
+				signatureVersion: "v4",
+				httpOptions: { timeout: 0 }
```

Comment:
I suggest adding `connectTimeout: 0` too. This should match closer the workaround we use for AWS CLI: `--cli-connect-timeout=0 --cli-read-timeout=0`.

Reply:
I've added `connectTimeout: 0` and performed another round of testing, which ended up being successful. I'll add it to the PR. Thanks for the suggestion, Kaloyan.